apps/E2E/src/TabList/pages/TabListPageObject.ts

```diff
@@ -0,0 +1,77 @@
+import { BasePage, By } from '../../common/BasePage';
+import { Attribute, AttributeValue } from '../../common/consts';
+import {
+  TABLIST_TESTPAGE,
+  HOMEPAGE_TABLIST_BUTTON,
+  TABLIST_TEST_COMPONENT,
+  FIRST_TAB,
+  SECOND_TAB,
+  THIRD_TAB,
+  FOURTH_TAB,
+  FIFTH_TAB,
+  TABLIST_CALLBACK_TEXT,
+} from '../consts';
+
+type Tab = 'First' | 'Second' | 'Third' | 'Fourth' | 'Fifth';
+
+class TabListPageObject extends BasePage {
+  async resetListSelection(): Promise<void> {
+    return (await this.getTab('First')).click();
+  }
+
+  async isTabSelected(selector: Tab): Promise<boolean> {
```

Comment:
I don't think we need to populate functions like this when it's just a wrapper over  this.compareAttribute(). It adds unnecessary technical debt.

It's perfectly valid to use .compareAttribute() in the spec file